lib/utils.js

```diff
@@ -23,6 +23,26 @@ var proxyaddr = require('proxy-addr');
 var qs = require('qs');
 var querystring = require('querystring');
 
+
+/**
+ * Helper function (requiring partial application)
+ * for setting exports.etag or exports.wetag.
+ *
+ * @param {Boolean} weak
+ * @return {Function}
+ * @api private
+ */
+
+function setTag(weak) {
+  return function(body, encoding) {
+    var buf = !Buffer.isBuffer(body)
+      ? new Buffer(body, encoding)
+      : body;
+
+    return etag(buf, {weak: weak});
```

Comment:
All net new code should be written in StandardJS style per https://github.com/expressjs/express/blob/master/Collaborator-Guide.md#prs-and-code-contributions